### PR TITLE
MANATI-VM: Configurable Sync folder from config.yml

### PR DESCRIPTION
**Pull request by rigoucr**

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -26,7 +26,7 @@ Vagrant.configure('2') do |config|
       type: 'nfs',
       create: true
     }
-    dev_config.vm.synced_folder '../www', '/home/vagrant/www', options
+    dev_config.vm.synced_folder vconfig['sync_folder_path'], '/home/vagrant/www', options
```
Comment (rigoucr) [inline Vagrantfile]: I don't know if is the right way, I think we could do this in other way with existing vagrant variables, but I thinks this is a useful feature for the VM.

Let me know  your opinion @kporras07

File: example.config.yml [hunk 2]
```diff
@@ -4,6 +4,12 @@
 vm_ram: "2048"
 vm_cpus: "2"
 
+#Sync Folder Config.
+
+# The local folder that will contains your sites.
```
Comment (kporras07) [inline example.config.yml]: *will contain

**Comment by kporras07** [comment]
@rigoucr it looks ok for me. Did you test this?
I'm also worried about people pulling from repo without updating config. It will explode.
Could you document this in README please?

**Comment by kporras07** [comment]
btw, synced please (instead of sync)

**Comment by rigoucr** [comment]
@kporras07 thanks for your comments, I added the suggested documentation

**Comment by kporras07** [comment]
Assuming Travis passes, :shipit:

**Pull request merged**